contracts/mainnet/connectors/wsteth/events.sol

```diff
@@ -0,0 +1,7 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.7.0;
+
+contract Events {
+    event LogDepositSTETH(uint256 tokenAmt, uint256 getId, uint256 setId);
```

Comment:
@shriyatyagii we can just keep LogDeposit and LogWithdraw

Reply:
Done